+dj/Schema.m

```diff
@@ -32,6 +32,7 @@
         %   computed: tableName with '__'
         allowedTiers = {'lookup' 'manual' 'imported' 'computed' 'job'}
         tierPrefixes = {'#', '', '_', '__', '~'}
+        tierClasses = {'dj.Lookup' 'dj.Manual' 'dj.Imported' 'dj.Computed' 'dj.Jobs'}
```

Comment:
That can be implemented that way once we completely transition to the new implementation.  The old implementation did not have corresponding classes and it is still being used.